pkg/kubectl/cmd/apply_test.go

```diff
@@ -220,7 +219,7 @@ func TestApplyNonExistObject(t *testing.T) {
 		Client: fake.HTTPClientFunc(func(req *http.Request) (*http.Response, error) {
 			switch p, m := req.URL.Path, req.Method; {
 			case p == pathNameRC && m == "GET":
-				return &http.Response{StatusCode: 404}, errors.NewNotFound("ReplicationController", "")
+				return &http.Response{StatusCode: 404, Body: ioutil.NopCloser(bytes.NewReader(nil))}, nil
```

Comment:
Ok. Just to confirm my understanding, the response returned by the APIServer will encode the error in the Body, so this problem won't occur, right?